libs/langchain/langchain/vectorstores/pgvector.py

```diff
@@ -132,7 +132,7 @@ def __post_init__(
         Initialize the store.
         """
         self._conn = self.connect()
-        # self.create_vector_extension()
+        self.create_vector_extension()
```

Comment:
also not sure why this was commented out, but seems like it should be part of the setup as well /cc @hwchase17

Reply:
without this, i get the following error:

```
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedObject) type "vector" does not exist
LINE 4:  embedding VECTOR,
                   ^

[SQL:
CREATE TABLE langchain_pg_embedding (
	collection_id UUID,
	embedding VECTOR,
	document VARCHAR,
	cmetadata JSON,
	custom_id VARCHAR,
	uuid UUID NOT NULL,
	PRIMARY KEY (uuid),
	FOREIGN KEY(collection_id) REFERENCES langchain_pg_collection (uuid) ON DELETE CASCADE
)

]
```

Reply:
The creation query doesn't raise an exception if it fails but rather a warning, proposed fix in #11930